packages/form-js-playground/test/spec/form.json

```diff
@@ -74,6 +74,57 @@
         }
       ]
     },
+    {
+      "key": "tags",
+      "label": "Taglist",
+      "type": "taglist",
+      "values": [

```

Comment:
Do we have a test for _(1.000 items +)_ tag list?

Reply:
We do not, but without dynamic data loading it's also not a covered scenario. Dynamic data might have me rethink some of the ways we handle values, so I would say let's wait on that PR (it's what I'm currently working on) to implement performance tests?

Reply:
Referring to the name of the closing issue (https://github.com/bpmn-io/form-js/issues/198) adding such a test right now would make sense to verify we really support this.

I would also be okay to wait for the dynamic data PR, but it's not necessary to wait for it. It should work regardless 👍

Reply:
> the dynamic data PR

In that case, probably the network will be the bottleneck and not the library.